bundle/artifacts/whl/autodetect.go

```diff
@@ -27,9 +27,9 @@ func (m *detectPkg) Name() string {
 }
 
 func (m *detectPkg) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
-	wheelTasks := libraries.FindAllWheelTasksWithLocalLibraries(b)
-	if len(wheelTasks) == 0 {
-		log.Infof(ctx, "No local wheel tasks in databricks.yml config, skipping auto detect")
+	tasks := libraries.FindTasksWithLocalLibraries(b)
```

Comment:
Should we return early if none of the libraries are Python wheels?

Reply:
No, not necessarily, python wheel libs can be used in other type of tasks (notebooks for example)

Reply:
Ah yes, that makes sense. But to be honest, it would still better to proceed trying to detect local python wheel package because it can be used in environment (not library section) and it would be a bit more difficult to detect